audit/src/main/java/gov/cms/ab2d/audit/job/FileDeletionJob.java

```diff
@@ -0,0 +1,47 @@
+package gov.cms.ab2d.audit.job;
+
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.FileTime;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.stream.Stream;
+
+@Slf4j
+public class FileDeletionJob implements Job {
+
+    public static final String EFS_MOUNT = "efsMount";
+
+    public static final String AUDIT_FILES_TTL_HOURS = "auditFilesTTLHours";
+
+    /**
+     * Delete all files that are in the efs mount that are older than the TTL variable
+     * @param jobExecutionContext
+     */
+    @Override
+    public void execute(JobExecutionContext jobExecutionContext) {
+        try (Stream<Path> walk = Files.walk(Paths.get(jobExecutionContext.getJobDetail().getJobDataMap().getString(EFS_MOUNT)))) {
```

Comment:
Let's build in some basic safety checks prior to/during the "walk". This is powerful code: imagine a developer sets `efs.mount` property to `/home/username` by mistake and then this job kicks in on their workstation...

Let's add code to ensure that:

- `efs.mount` value begins with `/` and is at least 5 characters long
- `efs.mount` does **not** start with any of the following: `/bin`,`/boot`,`/dev`,`/etc`,`/home`,`/lib`,`/opt`,`/root`,`/sbin`,`/sys`,`/usr`,`/Applications`,`/Library`,`/Network`,`/System`,`/Users`,`/Volumes`
- The walk _only_ deletes the files with `.ndjson` extension.